plugins/package-managers/node/src/funTest/kotlin/npm/NpmFunTest.kt

```diff
@@ -68,9 +68,9 @@ class NpmFunTest : WordSpec({
             patchActualResult(result.toYaml()) should matchExpectedResult(expectedResultFile, definitionFile)
         }
 
-        "resolve package-lock dependencies correctly" {
-            val definitionFile = getAssetFile("projects/synthetic/npm/package-lock/package.json")
-            val expectedResultFile = getAssetFile("projects/synthetic/npm/package-lock-expected-output.yml")
+        "resolve dependencies for a project with lockfile correctly" {
```

Comment:
Nit: Maybe the commit message scope should be "npm" instead of "node" as a whole?

Reply:
This is actually true for other commits in this PR as well. Now that all Node PMs are separate, it could be beneficial to specify more narrow scopes named after the concrete PM.